web/src/util/services/templates.ts

```diff
@@ -61,6 +61,16 @@ interface Manifest {
   REPLICATED_APP_URL: string;
   VELERO_VERSION: string;
   FLAGS: string;
+  INSTALLER_YAML: string;
+  INSTALLER_CRD: string;
+}
+
+function readCRDFromYAML(): string {
```

Comment:
This is the only additional functionality added to the TS api, don't know if there's some testing you think I should add around it.